Shuttle (New): NSF Heimdall - small detective shuttle

Tych0theSynth opened the pull request:
## About the PR
Adds the NSF Heimdall, a new small detective shuttle to replace the aging NSF Interceptor.
Includes the new wallmount radar that I recently added and was largely developed for this shuttle.
While the shuttle does look quite aggressive, I felt like detectives deserve to fly in style, you know?

This ship is intended to fill the same role as the Interceptor: small detective runabout, but perform the role much better, and have a more practical layout. I've also taken advantage of the new shelves to try and maximise storage space on top of any lockers already provided.

As per SOP, EVA and hardsuit are provided as part of the modest office space. 
A more open interior near the airlock leaves room for multiple persons to be on board and have a conversation without things being claustrophobic. 

Flight characteristics are good. Forward acceleration is plenty for the size of the ship, allowing for you to get from place to place quickly, as well as maintain speed while stalking prey. 

Key numbers:
Crew - 1
Tile count - 140 (15W x 16L) (Small)
Price - 27000 (appraises at around 22460 but a decent markup was slapped on thanks to the radar)

Designed by BlueBird Shipbuilding under contract by the New Frontier Sheriff's Department. 

## Why / Balance
The Interceptor, our beloved flying shoe, is rather tired now and has quite a number of problems. I tried my best to resolve these when I reworked it some time ago, but really it feels like it needs replacing with something that looks cooler and is overall better.

The Broadhead is really too large for a single det. This should hopefully allow for solo detectives to have a similar experience to that which they'd get on the larger ship. 

## How to test
Checkout the branch, purchase the ship.

## Media
![2024-11-15 19_40_50-Window](https://github.com/user-attachments/assets/8dbef855-8655-46f2-8b45-379fdcf895f2)
Initialised.

![2024-11-15 19_44_17-Window](https://github.com/user-attachments/assets/1dda619a-941f-4e30-b6d8-37c960d7d1c7)
Uninitialised.

![2024-10-28 12_52_52-Window](https://github.com/user-attachments/assets/ca384331-a609-47f6-a915-0ca5cf0dbc4c)
Radar profile next to the Broadhead.

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
Hopefully none!

**Changelog**
:cl:
- add: Added the NSF Heimdall, a new small detective shuttle.

Tych0theSynth committed:
Adds the NSF Heimdall

dvir001 commented:
If it will replace Interceptor please also delete it on the same PR please, might as well.

Tych0theSynth commented:
> If it will replace Interceptor please also delete it on the same PR please, might as well.

I was told specifically not to do this by a maptainer

Tych0theSynth committed:
Merge branch 'master' into heimdall

Jakumba commented:
This is really cool. I especially like the open plan interview section/office.

Tych0theSynth commented:
> Took a stroll through. Nice space fighter vibe. Feels cozier than you would think from the profile.
> 
> I was looking for a recharger, either for a sidearm or battery in general. Could be I missed it, but that might be a nice addition.
> 
> I'm not entirely familiar with detective equipment but do they generally want to have a spare box of forensic pads on hand?

Those can be easily purchased from the NFSD tech, and are available at spawn in the det closet on the NFSDO (they are also used very rarely).

And shite, I KNEW I forgot something when I was finishing it up the other day. I meant to add one.

Tych0theSynth committed:
Merge branch 'master' into heimdall

Tych0theSynth committed:
Added wallmount recharger

whatston3 committed:
Merge branch 'master' into heimdall

dvir001 committed:
Merge branch 'master' into heimdall

BlueHNT commented:
This design looks more like an attack fighter than a detective ship. Other than that, I love it.

Tych0theSynth commented:
> This design looks more like an attack fighter than a detective ship. Other than that, I love it.

Thank you! 

I won't deny the aggressive look, but why shouldn't detectives have something that looks cool to cruise around in? :D

Tych0theSynth committed:
Merge branch 'master' into heimdall

Tych0theSynth committed:
Merge branch 'master' into heimdall

Tych0theSynth committed:
Add holopad, remove EVA suit, fix shipyard file

Tych0theSynth commented:
![2025-02-01 17_20_20-MyServer - Frontier Station](https://github.com/user-attachments/assets/6e24bae6-424e-4503-9f8e-1366dff34b87)
Updated today. Moved some stuff around to make room for the holopad near the bridge, though this did mean I had to remove the SMES for it to feel accomodated properly. Not so amazing for power resilience, but it's a small shuttle anyway. 
Price was lowered a little as it appraises differently now. 
As we had clarification on SOP and hardsuits, the EVA locker is not required. It just has the patrol suit now.

Tych0theSynth committed:
Merge branch 'master' into heimdall

Tych0theSynth committed:
I perhaps forgot variantize

Tych0theSynth committed:
Merge branch 'master' into heimdall

Tych0theSynth committed:
move holopad, readd smes, adjust price

Cheackraze commented on Resources/Prototypes/_NF/Shipyard/Nfsd/heimdall.yml:
```diff
@@ -0,0 +1,42 @@
+  name: NSF Heimdall
+  parent: BaseVessel
+  description: A small detective runabout for intelligence gathering and forensic work.
+  price: 27000 # Approx 20% markup applied to the appraisal price of 22500 due to the presence of advanced radar.
```
nfsd ships are a bit more subjectively priced and I think this is acceptable

Cheackraze committed:
Merge branch 'master' into heimdall
